examples/spring-streams-example/src/test/kotlin/com/stove/spring/streams/example/e2e/TestSystemConfig.kt

```diff
@@ -0,0 +1,69 @@
+package com.stove.spring.streams.example.e2e
+
+import com.trendyol.stove.testing.e2e.bridge
+import com.trendyol.stove.testing.e2e.springBoot
+import com.trendyol.stove.testing.e2e.standalone.kafka.KafkaContainerOptions
+import com.trendyol.stove.testing.e2e.standalone.kafka.KafkaSystemOptions
+import com.trendyol.stove.testing.e2e.standalone.kafka.kafka
+import com.trendyol.stove.testing.e2e.system.TestSystem
+import kotlinx.coroutines.runBlocking
+import org.junit.jupiter.api.extension.BeforeAllCallback
+import org.junit.jupiter.api.extension.ExtensionContext
+import org.junit.jupiter.api.extension.ExtensionContext.Store.CloseableResource
+
+class Init :
```

Comment:
I suggest implementing Kotest#AbstractProjectConfig to get the benefit of coroutines and kotlin first test setup. As the code shows, you need bridge with `runBlocking`, and also need to track the lifetime with a `started` boolean.  

For example: https://github.com/Trendyol/stove/blob/main/examples/spring-example/src/test/kotlin/com/stove/spring/example/e2e/TestSystemConfig.kt#L12 

This implementation would give the same behaviour, in a clean way and more kotlin-way.

Reply:
done